classes/controller/AdminController.php

```diff
@@ -2614,6 +2614,7 @@ public function setMedia($isNewTheme = false)
 
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/theme.css', 'all', 0);
             $this->addJS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/main.bundle.js');
+            $this->addJS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/create_product.bundle.js');
```

Comment:
If this script is added in both themes, then this code should probably be added only once but outside of the `if/else` statement Also you should add a comment:

`// This is handled as an external common dependency for both themes, but once new-theme is the only one it should be integrated directly into the main.bundle.js file`

This will help future developers once we can get rid of the default theme